index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Hello World, we are software developers!</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="index.js"></script>
+    <nav id="navbar">
+      <a class="navlink" href="#welcome-section">About</a>
+      <a class="navlink" href="#projects">Work</a>
+      <a class="navlink" href="#profile">Profile</a>
+    </nav>
+    <div id="welcome-section">
+      <h1>Hey, i am Favour</h1>
+      <p class="wsp"><em>a software developer</em></p>
+    </div>
+
+    <div id="projects">
+      <h2 class="projects-header">My Projects</h2>
+
+      <div class="project-grid">
+        <a
+          class="projects project-title"
+          href="https://codepen.io/freeCodeCamp/full/zNqgVx"
+          target="_blank"
+        >
+          <img
+            class="images"
+            src="https://image.pbs.org/video-assets/eppEsWb-asset-mezzanine-16x9-LQ8tc0K.jpg.crop.1200x675.jpg"
+          />
+          <p class="project-ttle">Tribute Page</p>
+        </a>
+        <a
+          class="projects"
+          href="https://codepen.io/ritobrotomukherjee/details/xxpOQrq"
+          target="_blank"
+        >
+          <img
+            class="images"
+            src="https://image.pbs.org/video-assets/eppEsWb-asset-mezzanine-16x9-LQ8tc0K.jpg.crop.1200x675.jpg"
+          />
+          <p class="project-ttle">Tribute Page</p>
+        </a>
+        <a
+          class="projects"
+          href="https://codepen.io/ritobrotomukherjee/details/xxpOQrq"
+          target="_blank"
+        >
+          <img
+            class="images"
+            src="https://image.pbs.org/video-assets/eppEsWb-asset-mezzanine-16x9-LQ8tc0K.jpg.crop.1200x675.jpg"
+          />
+          <p class="project-ttle">Tribute Page</p>
+        </a>
+      </div>
+    </div>
+    <div id="profile">
+      <h1>Let's Work Together</h1>
+      <p class="wsp"><em>How do you take your coffee?</em></p>
+      <div class="links">
+        <a href="#">Facebook</a>
+        <a
+          id="profile-link"
+          target="_blank"
+          href="https://github.com/Favourezeugwa/FavourandRito"
+          >Github</a
+        >
+        <a id="profile-link" href="#">LinkedIn</a>
+      </div>
+    </div>
```

Comment:
I think this `<div>` tag for `id="profile"`  should be replaced with a `<footer>` as well. These are just proper HTML semantics guidelines. 😄